Peer.Domain/Commands/WatchArguments.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace Peer.Domain.Commands
+{
+    public class WatchArguments
+    {
+        public TimeSpan IntervalSeconds { get; } = TimeSpan.FromSeconds(30);
```

Comment:
Where the param isn't optional does this actually apply as a default?

Reply:
You're right this should just be done in the WatchOptions and not here. Result of moving things around a bit.